mitiq/pec/types/__init__.py

```diff
@@ -14,7 +14,7 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 from mitiq.pec.types.types import (
-    NoisyBasis,
     NoisyOperation,
     OperationRepresentation,
+    NoisyBasis,
 )
```

Comment:
Does changing the order of imports affect anything, or just reordering to show level of importance?

Reply:
This is because I first removed `NoisyBasis` from Mitiq and then re-added a deprecated version of the `NoisyBasis` later. Reordering was not intentional.

Reply:
To confirm, is the idea of re-adding just to warn the user that it's deprecated?

Reply:
Yes. We can completely remove it in 1 or 2 milestones.

Technically, this is not a _deprecation_ intended in the standard way, which would imply warning but still supporting `NoisyBasis`. In our case we just drop support for it and we give some info to the user with an error message.